libs/model/src/policies/ContestWorker.policy.ts

```diff
@@ -35,21 +35,21 @@ export function ContestWorker(): Policy<typeof inputs> {
         const fullContentUrl = getThreadUrl({
           chain: community!.id!,
           id: payload.id!,
-          title: payload.title,
         });
         // content url only contains path
         const contentUrl = new URL(fullContentUrl).pathname;
```

Comment:
this url functionality is key to match urls to content ids between on-chain contracts and platform... we should probably centralize all these utils in a single file to keep it consistent and reusable